admin/app/components/users/user-detail-personal-information/authentication-method.js

```diff
@@ -78,7 +77,7 @@ export default class AuthenticationMethod extends Component {
 
   get userOidcAuthenticationMethods() {
     return this.oidcIdentityProviders.list.map((oidcIdentityProvider) => {
-      const userHasThisOidcAuthenticationMethod = this.args.user.authenticationMethods.any(
+      const userHasThisOidcAuthenticationMethod = this.authenticationMethods.any(
```

Comment:
On a bcp de duplication. On aurait pu avoir une methode 
nitpick:
```suggestion
      hasAuthenticationMethod(identityProvider = 'PIX')
         this.authenticationMethods.any((authenticationMethod) => authenticationMethod.identityProvider === identityProvider)
      }
```